about.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="description" content="Browse Caitlin's abstract art">
+    <meta name="keywords" content="zentangles, abstract, art, drawing">
+
+    <title>Caitlin's Art</title>
+
+    <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <div id="container">
+      <div class="grey-background">
+        <a name="top"></a>
+        <a href="index.html"><img src="images/personal-logo-transparent.png" class="logo"></a>
+        <div class="navigation">
```

Comment:
Look into the `<nav>` element. Your decision on whether or not to use it but when there's a logical alternative we should think before using a `<div>`